lib/iris/cube.py

```diff
@@ -758,6 +767,231 @@ def _is_single_item(testee):
     return isinstance(testee, str) or not isinstance(testee, Iterable)
 
 
+class CubeAttrsDict(MutableMapping):
+    """
+    A dict-like object for "Cube.attributes", which provides unified user access to
+    the combined cube 'local' and 'global' attributes, mimicking the behaviour of a
+    simple dict.
+
+    This supports all the regular methods of a 'dict',
+    However, a few things such as the detailed print (repr) are different.
```

Comment:
```suggestion
    however, a few things such as the detailed print (repr) are different.
```
You end the above line with a comma

Reply:
Rather, replaced the comma with period.